package.json

```diff
@@ -11,6 +11,7 @@
     "watch": "broccoli-timepiece exports",
     "build": "./scripts/build.sh",
     "stats": "node scripts/size-calc",
+    "lint": "tslint src/**.ts",
```

Comment:
Should we run this command on test? To make sure all commits are checked automatically?

Reply:
added prettier🤘